Upgrade to Unity 2022 gives NativeList script error

I have a project that was working fine in Unity 2021. I want to upgrade it to 2022.

When I do I get the following error:

Library/PackageCache/com.community.netcode.transport.photon-realtime@d862504b14/Runtime/PhotonRealtimeTransport.cs(518,17): error CS0012: The type 'NativeList<>' is defined in an assembly that is not referenced. You must add a reference to assembly 'Unity.Collections, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.

The thing is that the Unity.Collections package IS installed in the project.

Can someone tell me how to fix this?

--- a/Transports/com.community.netcode.transport.photon-realtime/Runtime/com.community.netcode.transport.photon-realtime.asmdef
+++ b/Transports/com.community.netcode.transport.photon-realtime/Runtime/com.community.netcode.transport.photon-realtime.asmdef
@@ -2,7 +2,8 @@
     "name": "Photon Realtime Transport for Netcode for GameObjects",
     "rootNamespace": "Netcode.Transports.PhotonRealtime",
     "references": [
-        "Unity.Netcode.Runtime"
+        "Unity.Netcode.Runtime",
+        "Unity.Collections"
     ],
     "includePlatforms": [],
     "excludePlatforms": [],

